.github/workflows/main.yml

```diff
@@ -21,6 +21,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Install utils
+        run: |
+          if [ ${{ matrix.host }} = "macos-latest" ]; then
+            brew install coreutils
+          else
+            sudo apt-get update
+            sudo apt-get install -y coreutils
```

Comment:
The `timeout` utility lives in `coreutils`. It's usually preinstalled but let's just make sure it is.